packages/indicators/src/builders/buildArithmetic.ts

```diff
@@ -45,8 +49,20 @@ const fetchAnalyticClusters = async (
   const checkClusterIncludesAllElements = (cluster: AnalyticCluster) =>
     allElements.every(member => member in cluster.dataValues);
 
+  const replaceAnalyticValuesWithDefaults = (cluster: AnalyticCluster) => {
+    const returnDataValues = cluster.dataValues;
+    Object.keys(defaultValues).forEach(code => {
+      returnDataValues[code] = returnDataValues[code] || defaultValues[code];
```

Comment:
This will also replace 0's - be good to add a test case for this

Reply:
Now does not replace 0s, there is a test case for this.